public/index.html

```diff
@@ -30,6 +30,12 @@
     </script>
     <!-- End Google Analytics -->
 
+    <!-- Google Optmize (declaring gtag helper function) -->
+    <script>
+      function gtag() {if(dataLayer)dataLayer.push(arguments);}
```

Comment:
I've found:
```
window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
```
https://developers.google.com/gtagjs/devguide/snippet#google-analytics

Idk which one should be used